### PR TITLE
index.json update

### PR DESCRIPTION
**Pull request by limonspb**
api.betaflight.com not available yet from the Android due to api.betaflight.com "security issues" (need CORS). So we need to keep the Github index.json up-to-date till that gets fixed, hopefully by @blckmn .

### CODE REVIEW

**Review by butterSnake — APPROVED** [review]
:D

**Review by butterSnake — APPROVED** [review]
Lols

File: index.json [hunk 1]
```diff
@@ -92,57 +116,104 @@
...
       "filters",
       "freestyle",
       "freestyle_HD",
+      "freesytle",
       "frsky",
       "hd",
+      "hot",
       "jitter",
       "karate",
+      "karatreBrot",
```
Comment (haslinghuis) [inline index.json:195]:
```suggestion
      "karateBrot",
```

File: index.json [hunk 2]
```diff
@@ -92,57 +116,104 @@
...
       "names",
+      "nano",
+      "no rpm",
+      "noisy",
+      "normal",
+      "partial",
       "pid",
+      "pro32",
       "profile",
+      "quaddiction",
```
Comment (haslinghuis) [inline index.json:214]: Is this The RCAddict?

**Comment by haslinghuis** [comment]
Instead of putting al in one file growing and growing more and more, would it be possible to have it in separate files?

**Comment by haslinghuis** [comment]
We need to administer `Access-Control-Allow-Origin: URL`

**Comment by blckmn** [comment]
> We need to administer `Access-Control-Allow-Origin: URL`

This is now sorted :)

**Review by blckmn — APPROVED** [review]

**Review by sugaarK — COMMENTED** [review]
@limonspb is this sorted now?

File: index.json (hunk 1 above)
Comment (limonspb) [reply index.json:195]: that's in @sugaarK  preset
index.json is generated automatically.

File: index.json (hunk 2 above)
Comment (limonspb) [reply index.json:214]: same, index.json is generated automatically.
So that's again a question to @sugaarK  if hr wants to create a PR to fix that.

**Comment by limonspb** [comment]
> Instead of putting al in one file growing and growing more and more, would it be possible to have it in separate files?

It is not all in one. Index.json contains only headers of the presets. The size of this file is roughly 1kb per preset.
By the time we have 1000 presets this file will be only about 1mb.

For 10.9 we should make a new preset source - online SQL database. But for the next few years, we are pretty safe with files.

**Comment by limonspb** [comment]
> @limonspb is this sorted now?

yes don't need to merge this anymore, thanks!

File: index.json (hunk 2 above)
Comment (sugaarK) [reply index.json:214]: Name of our local club.. it’s spelt correct

**Comment by limonspb** [comment]
not needed as api.betaflight.com is fixed now

**Pull request closed without merging**